### PR TITLE
Update help window

### PR DESCRIPTION
**Pull request by dlimyy**
Updated help window to include all commands.

### CODE REVIEW

File: src/main/resources/view/HelpWindow.fxml [hunk 1]
```diff
@@ -19,26 +18,160 @@
...
+                                      <Label style="-fx-padding: 0 0 0 20;" text="T del INDEX" textFill="WHITE" />
+                                      <Label prefHeight="17.0" prefWidth="411.0" style="-fx-padding: 0 0 0 222;" text="Deletes the task in the task list" textFill="WHITE" />
+                                  </HBox>
+                                  <HBox prefHeight="38.0" prefWidth="495.0">
+                                      <Label style="-fx-padding: 0 0 0 20;" text="T edit INDEX [m/MODULE] [d/DESCRIPTION]" textFill="WHITE" />
+                                      <Label style="-fx-padding: 0 0 0 43;" text="Edits the task in the task list" textFill="WHITE" />
+                                  </HBox>
+                                  <HBox prefHeight="36.0" prefWidth="495.0">
+                                      <Label style="-fx-padding: 0 0 0 20;" text="T list" textFill="WHITE" />
+                                      <Label style="-fx-padding: 0 0 0 260;" text="Lists the task in the task list" textFill="WHITE" />
```
Comment (RachelChua) [inline src/main/resources/view/HelpWindow.fxml:55]: Should this be 'tasks'. Same for sorting below.

Comment (dlimyy) [reply src/main/resources/view/HelpWindow.fxml:55]: Yes i agree I shall change it

File: src/main/resources/view/HelpWindow.fxml [hunk 2]
```diff
@@ -19,26 +18,160 @@
...
+                                    <Label style="-fx-padding: 0 0 0 181;" text="To exit MODPRO application" textFill="WHITE" />
+                                </HBox>
+                            </VBox>
+                        </children>
+                     </HBox>
+                     <HBox fx:id="helpLinkContainer" alignment="CENTER" prefHeight="80.0" prefWidth="1000.0">
+                        <children>
+                           <Label fx:id="helpMessage" style="-fx-padding: 0 15 0 0;" text="helpMessage" />
+                           <Button fx:id="copyButton" mnemonicParsing="false" onAction="#copyUrl" text="Copy URL" />
+                        </children></HBox>
```
[the remark this answers is not in the document]
Comment (dlimyy) [reply src/main/resources/view/HelpWindow.fxml:171]: Ok I will make these changes

**Review by Sampy147 — APPROVED** [review]
LGTM!

**Pull request merged**